plugins/rtmp-services/data/services.json

```diff
@@ -1394,11 +1394,11 @@
             }
         },
         {
-            "name": "Madcat",
+            "name": "Trovo",
```

Comment:
If you are changing the name of the service, you must also include the previous names so that old configurations correctly update to the new name. An example of how to do that is at https://github.com/obsproject/obs-studio/blob/9130a1bc28899af0dbfae2a79a6e05bf2494b90c/plugins/rtmp-services/data/services.json#L1266

Reply:
Done. Added old name "Madcat" in alt_names.